### PR TITLE
Add build system and images for sdn

### PR DESCRIPTION
**Pull request by tnozicka**
/cc @deads2k @sttts 

I am gonna open one over to the published repo to setup a CI and prove it works.

### CODE REVIEW

**Comment by tnozicka** [comment]
CI PR https://github.com/openshift/release/pull/4189

**Comment by tnozicka** [comment]
sdn repo PR https://github.com/openshift/sdn/pull/1

**Comment by tnozicka** [comment]
https://github.com/openshift/sdn/pull/1 show the new CI is green

Not to confuse it with total success the images build over there are not used yet. We need to:
1. Merge this PR
2. Wait for it to be published
3. Migrate over from sdn images build in origin to images build from the published repo


`3.` Is where the transition is not atomic yet - we would either need to hookup CI so it restores vendor in every job (not trivial change) or make publishing bot call `glide update --strip-vendor` before publishing

File: staging/src/github.com/openshift/sdn/glide.yaml [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+- package: github.com/openshift/source-to-image
+  version: master
+- package: github.com/openshift/oauth-server
+  version: master
+- package: github.com/openshift/oc
+  version: master
+- package: github.com/openshift/openshift-controller-manager
+  version: master
+- package: github.com/openshift/openshift-apiserver
+  version: release-4.2
```
Comment (sallyom) [inline staging/src/github.com/openshift/sdn/glide.yaml]: this should be master? i get a glide error w/ release-4.2

**Comment by deads2k** [comment]
I think we got past this

/close

**Pull request closed without merging**